extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -448,7 +440,7 @@ export class AmpStoryPage extends AMP.BaseElement {
     );
     return Promise.all([
       this.beforeVisible(),
-      this.mediaLayoutPromise_,
+      this.waitForMediaLayout_,
```

Comment:
Could you test if this works - through a unit test or anything else?
You're not calling the method here, it's not waiting for the returned promise, which makes me think that this is super easy to break and very hard to notice. :(

Reply:
Done.